src/main/php/PDepend/Metrics/Analyzer/NPathComplexityAnalyzer.php

```diff
@@ -236,11 +233,10 @@ public function visitDoWhileStatement(ASTDoWhileStatement $node): void
         $this->dispatch($node->getChild(0));
         $expr = $this->sumComplexity($node->getChild(1));
 
-        $npath = MathUtil::add($expr, $this->collector);
-        $npath = MathUtil::add($npath, '1');
+        $npath = $expr + $this->collector + 1;
```

Comment:
I would go with the collector always at the beginning:

```suggestion
        $npath = $this->collector + $expr + 1;
```

Reply:
Done (also the other case)